From 5e7c83ad0b352791e175f626b35bc81eed8b805e Mon Sep 17 00:00:00 2001
From: Vlad Yasevich <vladislav.yasevich@hp.com>
Date: Thu, 12 Nov 2009 10:38:44 -0500
Subject: [PATCH 10/53] API: Add SCTP_SACK_IMMEDIATELY defintion.

---
 src/include/netinet/sctp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/include/netinet/sctp.h b/src/include/netinet/sctp.h
index bb0bd0b..99606d1 100644
--- a/src/include/netinet/sctp.h
+++ b/src/include/netinet/sctp.h
@@ -208,6 +208,7 @@ enum sctp_sinfo_flags {
 	SCTP_UNORDERED = 1,  /* Send/receive message unordered. */
 	SCTP_ADDR_OVER = 2,  /* Override the primary destination. */
 	SCTP_ABORT=4,        /* Send an ABORT message to the peer. */
+	SCTP_SACK_IMMEDIATELY = 8,      /* SACK should be sent without delay */
 	SCTP_EOF=MSG_FIN,    /* Initiate graceful shutdown process. */
 };
 
-- 
1.7.11.7

